plugins/playlist/src/components/PlaylistPage/PlaylistEntitiesTable.tsx

```diff
@@ -73,16 +74,22 @@ export const PlaylistEntitiesTable = ({
   const removeEntity = useCallback(
     async (_, entity: Entity | Entity[]) => {
       try {
+        const entityName = [entity].flat().map(item => item.metadata.name);
         await playlistApi.removePlaylistEntities(
           playlistId,
           [entity].flat().map(stringifyEntityRef),
         );
         loadEntities();
+        alertApi.post({
+          message: `Removed entity '${entityName}' from table`,
```

Comment:
Small wording change just in case we display entities in a non-table component in the future
```suggestion
          message: `Removed entity '${entityName}'`,
```